saleor/cart/views.py

```diff
@@ -40,12 +40,21 @@ def index(request, cart):
         cart_total = cart.get_total(discounts=discounts)
         local_cart_total = to_local_currency(cart_total, request.currency)
 
+    default_country = get_default_country(request)
+    country_form = CountryForm(request.POST or None,
+                               initial={'country': default_country})
+
+    if country_form.is_valid():
+        shipments = country_form.get_shipment_options()
+        return JsonResponse({'options': list(shipments)})
```

Comment:
It may be useful to create separated view that will return shipping options. Cart index view should be responsible only for rendering cart content.
Also consider using queryset's `values()` method to return only values that you really need.